src/Physics/Multinucleon/XSection/NievesSimoVacasMECPXSec2016.cxx

```diff
@@ -339,4 +352,21 @@ void NievesSimoVacasMECPXSec2016::LoadConfig(void)
         dynamic_cast<const XSecIntegratorI *> (
           this->SubAlg("NumericalIntegrationAlg"));
         assert(fXSecIntegrator);
+
+	bool good_config = true ; 
+	// Read optional MECScaleVsW:
+	fMECScaleVsW = nullptr; 
+	if( GetConfig().Exists("MECScaleVsWAlg") ) {
+	  fMECScaleVsW = dynamic_cast<const MECScaleVsW *> ( this->SubAlg("MECScaleVsWAlg") );
+	  if( !fMECScaleVsW ) {
+	    good_config = false ; 
+	    LOG("NievesSimoVacasMECPXSec2016", pERROR) << "The required MECScaleVsWAlg does not exist. AlgID is : " << SubAlg("MECScaleVsWAlg")->Id() ;
```

Comment:
Wrong message. It's not that the ago does not exist, it's that it cannot be casted

Reply:
meaning it's not of the desired type